docs/02.API-REFERENCE.md

```diff
@@ -2322,6 +2335,79 @@ main (void)
 - [jerry_create_container](#jerry_create_container)
 - [jerry_container_type_t](#jerry_container_type_t)
 
+*New in version [[NEXT_RELEASE]]*.
+
+## jerry_contaiter_operation
+
+**Summary**
+
+Perform container operation on the given operands (add, delete, set, etc.).
+
+*Note*:
+- Returned value must be freed with [jerry_release_value](#jerry_release_value) when it
+  is no longer needed.
+- This API function depends on a build option (`JERRY_BUILTIN_CONTAINER`) and can be checked
+  runtime with the `JERRY_FEATURE_MAP, JERRY_FEATURE_SET, JERRY_FEATURE_WEAKMAP, JERRY_FEATURE_WEAKSET`
```

Comment:
So... this method works on weaksets?

Reply:
It should work on them. Also:
`JERRY_FEATURE_MAP, JERRY_FEATURE_SET, JERRY_FEATURE_WEAKMAP, JERRY_FEATURE_WEAKSET` ->
`JERRY_FEATURE_MAP`, `JERRY_FEATURE_SET`, `JERRY_FEATURE_WEAKMAP`, `JERRY_FEATURE_WEAKSET`